sasl/base/src/main/java/org/wildfly/security/sasl/WildFlySasl.java

```diff
@@ -164,5 +164,12 @@
      * Note: This is a server only property and is not used client side.
      */
     public static final String GSSAPI_CREATE_NAME_GSS_INIT = "wildfly.sasl.gssapi.server.create-name-gss-init";
+    
+    /**
+     * A property used to disable Unicode normalization for passwords.
+     *
+     * Note: This is a server only property and is not used client side.
```

Comment:
I think this note can be removed since the property gets used on the client side in ```PlainSaslClient```.

Reply:
done